lib/translate/common.h

```diff
@@ -15,10 +15,16 @@
 #ifndef LIB_TRANSLATOR_COMMON_H
 #define LIB_TRANSLATOR_COMMON_H
 
+namespace translator {
+
+enum class TranslatorStatusCode { SUCCESS, FAILURE };
```

Comment:
```suggestion
enum class StatusCode { SUCCESS, FAILURE };
```

Avoid repeating the namespace in the name; downstream code that wants/needs to be explicit can qualify with the namespace as desired.

Reply:
I'll also make this change in https://github.com/googleinterns/scsi2nvme/pull/62